### ISSUE TITLE
WCAG 4.1.2: Ensure every ARIA toggle field has an accessible name

### ISSUE THREAD

**Issue by mgifford** [twentyhq/twenty#23130]
Title: WCAG 4.1.2: Ensure every ARIA toggle field has an accessible name (#base-ui-_r_1e_)
Tags: Accessibility, WCAG 4.1.2, aria-toggle-field-name

Issue: Ensure every ARIA toggle field has an accessible name (aria-toggle-field-name - https://accessibilityinsights.io/info-examples/web/aria-toggle-field-name)

Target application: All Companies - Companies - https://crm.civicactions.com/objects/companies?viewId=242124e5-2d4e-4fb4-bf1a-8f50d473879d

Element path: #base-ui-_r_1e_

Snippet: <span data-unchecked="" role="checkbox" tabindex="0" id="base-ui-_r_1e_" aria-checked="false" data-testid="input-checkbox" class="_root_1ph8w_1 _primary_1ph8w_17 _small_1ph8w_30 _squared_1ph8w_41 _blue_1ph8w_52 _hoverable_1ph8w_64"><span class="_box_1ph8w_102"></span></span>

How to fix: 
Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Environment: Chrome version 150.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.47.0 (axe-core 4.11.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
